parity/lib.rs

```diff
@@ -241,10 +241,5 @@ pub fn start<Cr, Rr>(
 		Cr: Fn(String) + 'static + Send,
 		Rr: Fn() + 'static + Send
 {
-	let deprecated = find_deprecated(&conf.args);
```

Comment:
How is does deprecation work in this new version? It'd be good to have some internal documentation on how to best add/deprecate/remove cli flags/options/subcommands somewhere.